lib/hpcbase.pm

```diff
@@ -4,15 +4,6 @@ use strict;
 use testapi;
 use utils 'systemctl';
 
-sub exec_and_insert_password {
```

Comment:
For lazy readers : looks fine :+1:  :)

I agree with you that generally speaking enable_and_start is also not actually hpc specific and could be placed in utils.pm . But this is the problem of utils.pm too many things looks good enough for utils.pm and there are too many reasons why having files over 1000 lines is bad thing . Not saying that we should stop adding functions to utils.pm but let's at least do this with really good reason and think twice "where else I can put it ? ".